Sakura/database/cache.py

```
@@ -0,0 +1,221 @@
+    except Exception as e:
+        logger.error(f"❌ Failed to initialize Valkey client: {e}")
+        valkey_client = None
+        return False
+
+async def close_valkey():
+    """Close Valkey connection"""
+    if valkey_client:
+        try:
+            await valkey_client.aclose()
+            logger.info("✅ Valkey connection closed")
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)